lms/templates/fields/field_image.underscore

```diff
@@ -1,17 +1,17 @@
 <div class="image-wrapper">
-    <img class="image-frame" src="<%- imageUrl %>"  alt="<%=imageAltText%>"/>
+    <img class="image-frame" src="<%- imageUrl %>"  alt="<%= HtmlUtils.ensureHtml(imageAltText) %>"/>
```

Comment:
This is another example of a very generic template being used in multiple instances.